From f66ab8242826049aa999589fc614ebedd7dd9ed8 Mon Sep 17 00:00:00 2001
From: IMX6 DEV <imx6@imx6-virtual-machine.(none)>
Date: Tue, 19 Apr 2016 16:01:06 +0800
Subject: [PATCH 9/9] mkspec: Fix linux-oe-g-platform patch
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

oe-device-extra.priがない場合がある
Error: Cannot read /opt/poky/1.7/sysroots/cortexa9hf-vfp-neon-poky-linux-gnueabi/usr/lib/qt5/mkspecs/oe-device-extra.pri: No such file or directory
---
 mkspecs/linux-oe-g++/qmake.conf |    4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/mkspecs/linux-oe-g++/qmake.conf b/mkspecs/linux-oe-g++/qmake.conf
index 311ba04..de279d2 100644
--- a/mkspecs/linux-oe-g++/qmake.conf
+++ b/mkspecs/linux-oe-g++/qmake.conf
@@ -37,6 +37,8 @@ QMAKE_LINK_C_SHLIB = $(OE_QMAKE_LINK)
 # for the SDK
 isEmpty(QMAKE_QT_CONFIG):QMAKE_QT_CONFIG = $(OE_QMAKE_QT_CONFIG)
 
-include(../oe-device-extra.pri)
+exists(../oe-device-extra.pri) {
+    include(../oe-device-extra.pri)
+}
 
 load(qt_config)
-- 
1.7.9.5

